packages/tupaia-web/src/components/Alert.tsx

```diff
@@ -0,0 +1,95 @@
+/*
+ * Tupaia
+ * Copyright (c) 2017 - 2020 Beyond Essential Systems Pty Ltd
+ */
+
+import React from 'react';
+import MuiAlert, { AlertProps } from '@material-ui/lab/Alert';
+import styled from 'styled-components';
+import { CheckCircle, Warning } from '@material-ui/icons';
+
+const StyledSmallAlert = styled(MuiAlert)`
```

Comment:
Can we use the Alert component that's in ui-components. It doesn't matter if it's not 100% the same as the alert in the old site as the alerts are pretty inconstant any way. Or we could extend the ui-components ones if need be. also I would prefer to just use the material-ui theme for the colors. We can set alert colors in the theme if need be but I'm sure the error and success ones are already pretty good.